src/components/AllServices/allServicesLinks.ts

```diff
@@ -12,15 +12,16 @@ export type AllServicesGroup = {
   isGroup: true;
   title: string;
   ITLess?: boolean;
-  links: AllServicesLink[];
+  links: (string | AllServicesLink)[];
   description?: string;
 };
 export type AllServicesSection = {
-  icon: keyof typeof AllServicesIcons;
+  id?: string;
+  icon?: keyof typeof AllServicesIcons;
   ITLess?: boolean;
   title: string;
-  description: string;
-  links: (AllServicesLink | AllServicesGroup)[];
+  description?: string;
+  links: (string | AllServicesLink | AllServicesGroup)[];
```

Comment:
When can we get links as string? Each link requires both pathname and title

Reply:
The services config from chrome service holds the link information as strings to reference the real thing `rhel.inventory` will be translated to 
```
{
            "id": "inventory",
            "appId": "inventory",
            "title": "Inventory",
            "href": "/insights/inventory",
            "product": "Red Hat Insights",
            "description": "View details about your Red Hat Enterprise Linux systems."
          },
```